sdk/communication/AzureCommunicationChat/AzureCommunicationChat.xcodeproj/project.pbxproj

```diff
@@ -448,7 +448,7 @@
 		D15E98E025B7C80C00E63DB2 /* Models */ = {
 			isa = PBXGroup;
 			children = (
-				D17C7B5225FBF4D5007AD39B /* ChatError.swift */,
+				0A8B9FF4264C6D38009C24B0 /* CommunicationError.swift */,
```

Comment:
Did we revert back to CommunicationError? @Leoaqr @tjprescott

Reply:
I think I need the autorest directive you used. It's not in my readme.

Reply:
Oh gotcha, original directive we used:
```
# Rename CommunicationError to ChatError
- from: swagger-document
  where: '$.definitions.CommunicationError'
  transform: >
    $["x-ms-client-name"] = "ChatError";
```

Or with your remap `--remap-models CommunicationError=ChatError`

Reply:
If there is autorest changes ready we can help regenerate too

Reply:
I used the Autorest directive. Using the flag (renamed to `--generate-as-internal`) would make it `internal` which we don't want.